app/invocation/invocation_action_card.tsx

```diff
@@ -148,11 +146,16 @@ export default class InvocationActionCardComponent extends React.Component<Props
       .fetchBytestreamFile(inputRootURL, this.props.model.getInvocationId(), "arraybuffer")
       .then((buffer) => {
         let inputRoot = build.bazel.remote.execution.v2.Directory.decode(new Uint8Array(buffer));
-        let inputDirs: InputNode[] = inputRoot.directories.map((node) => ({
+        let inputNodes: TreeNode[] = inputRoot.directories.map((node) => ({
           obj: node,
           type: "dir",
         }));
-        this.setState({ inputRoot, inputDirs });
+        let inputSymlinks: TreeNode[] = inputRoot.symlinks.map((node) => ({
+          obj: node,
+          type: "symlink",
+        }));
+        inputNodes.push(...inputSymlinks);
```

Comment:
nit: avoid calling functions using spread args because it will result in a stack overflow if you try to pass too many args.

```suggestion
        const inputNodes = [...inputDirectories, ...inputSymlinks];
```